api/src/service/cache2-backwards-compatibility.spec.ts

```diff
@@ -130,6 +130,36 @@ const workflowitemStreamItems: any[] = [
           documents: [],
           permissions: {},
           additionalData: {},
+          type: "general",
```

Comment:
Is it type or workflowitemType? I think one should be deleted to test one with and one without type

Reply:
right one should be without type, I already noticed, I made a mistake when I rebased